src/pages/api/data-requests.js

```diff
@@ -0,0 +1,59 @@
+import { submitSlackDataRequest } from 'api/data-requests/slack'
+
+export default async (req, res) => {
+  // requestValues should include form values, query/accessionCode, and the request type ('experiment' or 'search')
```

Comment:
let's just have a commented out example version of the post data in this comment.

Reply:
See the commit at 1c555da48a1276e1e3c3b40e9f4fe0109ee3d27e.